docs/protocol/avs.mdx

```diff
@@ -0,0 +1,47 @@
+# Hyperlane AVS
+
+### Protocol SLA
```

Comment:
I would either be a lot more elaborate or remote this page. Right now, as somebody who doesn't know anything about the Hyperlane AVS, I have no idea about the purpose of the AVS, what a challenger is, etc.

Reply:
Ok, I'll remove this and add this section to a new PR where I'll add more details. Don't want to block operators looking at the guide.